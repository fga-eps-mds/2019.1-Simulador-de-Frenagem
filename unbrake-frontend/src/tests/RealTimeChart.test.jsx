// react-test-render

// import React from "react";
import Enzyme from "enzyme";
// import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
/*
 * import { Provider } from "react-redux";
 * import { createStore } from "redux";
 * import { create } from "react-test-renderer";
 * import RealTimeChart from "../components/RealTimeChart";
 * import reducers from "../reducer/index";
 */

Enzyme.configure({ adapter: new Adapter() });

describe("<RealTimeChart />", () => {
  describe("render()", () => {
    test("renders the component", () => {
      /*
       * const context = React.createContext(RealTimeChart);
       * const store = createStore(reducers);
       * const wrapper = create(
       *   <Provider store={store}>
       *     <RealTimeChart store={store} />
       *   </Provider>
       *   // { context: store }
       * );
       * const component = wrapper.dive();
       */
      // expect(toJson(component)).toMatchSnapshot();
    });
  });
});
