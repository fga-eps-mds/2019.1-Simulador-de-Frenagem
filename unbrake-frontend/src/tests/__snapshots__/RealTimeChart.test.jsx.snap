// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RealTimeChart /> render() renders the component 1`] = `
<ChartComponent
  data={
    Object {
      "datasets": Array [
        Object {
          "backgroundColor": "#305c8a",
          "borderColor": "#305c8a",
          "cubicInterpolationMode": "monotone",
          "data": Array [],
          "fill": false,
          "label": "Depth",
        },
      ],
    }
  }
  datasetKeyProvider={[Function]}
  height={150}
  legend={
    Object {
      "display": true,
      "position": "bottom",
    }
  }
  options={
    Object {
      "hover": Object {
        "intersect": false,
        "mode": "nearest",
      },
      "legend": Object {
        "display": false,
      },
      "maintainAspectRatio": false,
      "scales": Object {
        "xAxes": Array [
          Object {
            "gridLines": Object {
              "display": true,
            },
            "maxBarThickness": 3,
            "realtime": Object {
              "delay": 1000,
              "duration": 20000,
              "onRefresh": [Function],
              "refresh": 3000,
            },
            "type": "realtime",
          },
        ],
        "yAxes": [Function],
      },
      "title": Object {
        "display": false,
      },
      "tooltips": Object {
        "enabled": false,
        "intersect": false,
        "mode": "nearest",
      },
    }
  }
  redraw={false}
  type="line"
  width={300}
/>
`;
